app/src/main/java/com/naman14/timberx/ui/adapters/FolderAdapter.kt

```diff
@@ -135,37 +130,34 @@ class FolderAdapter(private val mContext: Activity) : RecyclerView.Adapter<Folde
 
         override fun onPostExecute(files: List<File>) {
             super.onPostExecute(files)
-            mFileSet = files
+            this@FolderAdapter.files = files
             notifyDataSetChanged()
-            mBusy = false
+            isBusy = false
             prefs.edit {
-                putString(LAST_FOLDER, mRoot!!.path)
+                putString(KEY_LAST_FOLDER, rootFolder!!.path)
             }
         }
     }
 
     inner class ItemHolder(view: View) : RecyclerView.ViewHolder(view), View.OnClickListener {
-
-        var title: TextView
-        var albumArt: ImageView
+        var title = view.findViewById<TextView>(R.id.folder_title)!!
```

Comment:
Why not `val` and why the `!!`?

Reply:
@Zhuinden Could be `val`. `!!` because the Android framework doesn't use `@NonNull` so Kotlin gives warnings with this.